e2e-tests/tests/features/dbbackup_pxc/81-delete-restore.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: kuttl.dev/v1
+kind: TestStep
+timeout: 10
+commands:
+  - command: kubectl -n "${NAMESPACE} dbr --all"
+  - command: kubectl -n "${NAMESPACE} pxc-restore --all"
```

Comment:
a bit confused, what is the purpose of this commands?

Reply:
it's a bug. It should have included `delete`